src/components/PeopleTable.tsx

```diff
@@ -1,682 +1,171 @@
-              className="has-text-danger"
-              href="#/people/elisabeth-haverbeke-1711"
-            >
-              Elisabeth Haverbeke
-            </a>
-          </td>
-          <td>f</td>
-          <td>1711</td>
-          <td>1754</td>
-          <td>
-            <a
-              className="has-text-danger"
-              href="#/people/maria-de-rycke-1683"
-            >
-              Maria de Rycke
-            </a>
-          </td>
-          <td>
-            <a href="#/people/jan-haverbeke-1671">Jan Haverbeke</a>
-          </td>
-        </tr>
+export const PeopleTable:React.FC<Prop> = ({
+  people,
+}) => {
+  const [searchParams] = useSearchParams();
+  const tableSortingNames = ['Name', 'Sex', 'Born', 'Died'];
+  const { slug } = useParams();
 
-        <tr data-cy="person">
-          <td>
-            <a href="#/people/jan-van-brussel-1714">Jan van Brussel</a>
-          </td>
-          <td>m</td>
-          <td>1714</td>
-          <td>1748</td>
-          <td>Joanna van Rooten</td>
-          <td>Jacobus van Brussel</td>
-        </tr>
+  const sort = searchParams.get('sort');
+  const order = searchParams.get('order');
 
-        <tr data-cy="person">
-          <td>
-            <a href="#/people/bernardus-de-causmaecker-1721">
-              Bernardus de Causmaecker
-            </a>
-          </td>
-          <td>m</td>
-          <td>1721</td>
-          <td>1789</td>
-          <td>
-            <a
-              className="has-text-danger"
-              href="#/people/livina-haverbeke-1692"
-            >
-              Livina Haverbeke
-            </a>
-          </td>
-          <td>
-            <a href="#/people/lieven-de-causmaecker-1696">
-              Lieven de Causmaecker
-            </a>
-          </td>
-        </tr>
+  const personLink = (name: string | null) => {
+    const foundPerson = people.find(person => person.name === name);
 
-        <tr data-cy="person">
-          <td>
-            <a href="#/people/jan-francies-haverbeke-1725">
-              Jan Francies Haverbeke
-            </a>
-          </td>
-          <td>m</td>
-          <td>1725</td>
-          <td>1779</td>
-          <td>Livina de Vrieze</td>
-          <td>
-            <a href="#/people/pieter-bernard-haverbeke-1695">
-              Pieter Bernard Haverbeke
-            </a>
-          </td>
-        </tr>
+    return foundPerson ? foundPerson.slug : null;
+  };
 
-        <tr data-cy="person">
-          <td>
-            <a
-              className="has-text-danger"
-              href="#/people/angela-haverbeke-1728"
-            >
-              Angela Haverbeke
-            </a>
-          </td>
-          <td>f</td>
-          <td>1728</td>
-          <td>1734</td>
-          <td>Livina de Vrieze</td>
-          <td>
-            <a href="#/people/pieter-bernard-haverbeke-1695">
-              Pieter Bernard Haverbeke
-            </a>
-          </td>
-        </tr>
+  const sortedPeople = () => {
```

Comment:
it's not a good practice to put true as switch condition.
you can use sort value instead.

and consider using [destructuring assignment](https://javascript.info/destructuring-assignment#array-destructuring) for swapping variables if there is desc order. lide this:

```typescript
let [personA, personB] = [a, b];

if (order === 'desc') {
  [personA, personB] = [personB, personA]
}
```

Reply:
and don't forget to make a copy since sort is mutating method

Reply:
and you can put switch case inside sort method to avoid code duplicates